### PR TITLE
RAINCATCH-354 [raincatcher-demo-mobile] - When entering a form tab and there is a no data informing it to the user. Example: "Data not found"

### PR DESCRIPTION
**Pull request by camilamacedo86**

### CODE REVIEW

File: src/app/appform/empty.tpl.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="empty-state" layout-padding layout-margin>
+  <h2 class="md-title">No form associated.</h2>
```
Comment (nialldonnellyfh) [inline src/app/appform/empty.tpl.html]: Probably: "No forms available."

File: src/app/appform/empty.tpl.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+  <p class="md-body-1">If you use here forms from Red Hat Mobile Application Platform, go there and associate this project with one or more forms.</p>
```
Comment (nialldonnellyfh) [inline src/app/appform/empty.tpl.html]: Something like: "If you want to use the Drag & Drop Apps Forms feature of the Red Hat Mobile Application Platform, associate this project with one or more forms" maybe?

**Review by nialldonnellyfh — COMMENTED** [review]
Thanks @camilamacedo86 , couple of small comments on the messages.

Also, can you squash your commits.

**Comment by camilamacedo86** [comment]
@nialldonnellyfh the changes in the texts was made as your suggestions.

**Comment by camilamacedo86** [comment]
@nialldonnellyfh one question. We have here: Some checks were not successful with "security/snyk — 29 new vulnerable dependency paths" However, my change is only 2 html files :( Could you help me understand it? Is it a new validation ?

**Comment by nialldonnellyfh** [comment]
@camilamacedo86 Can you rebase the branch from master. It shouldn't be failing

**Comment by camilamacedo86** [comment]
Done:-) @nialldonnellyfh Tks for your help and answer :-)

**Comment by nialldonnellyfh** [comment]
@camilamacedo86 Looks good. I can squash and merge if you like.

Thanks a lot for the PR!!

**Comment by camilamacedo86** [comment]
I get was is required. For the next ones, I will squash and send all in one commit. If you think it will be easier for you to do since you will merge please feel free. I am learning the process .. because this I am getting only small things for feeling comfortable with it first.

**Pull request merged**